src/Writers/HTMLWriter.jl

```diff
@@ -1748,7 +1749,17 @@ function mdconvert(d::Dict{MIME,Any}, parent; kwargs...)
     if haskey(d, MIME"text/html"())
         out = Documents.RawHTML(d[MIME"text/html"()])
     elseif haskey(d, MIME"image/svg+xml"())
-        out = Documents.RawHTML(d[MIME"image/svg+xml"()])
+        svg = d[MIME"image/svg+xml"()]
+        # the xmlns attribute has to be present for data:image/svg+xml
+        # to work (https://stackoverflow.com/questions/18467982)
+        # URIs.escapeuri is needed to replace all special characters from the SVG string
+        # which could break the HTML
+        out = if occursin("xmlns=", svg)
+            escaped_svg = URIs.escapeuri(svg)
+            Documents.RawHTML(string("<img src=\"data:image/svg+xml,", escaped_svg, "\" />"))
+        else
+            Documents.RawHTML(d[MIME"image/svg+xml"()])
```

Comment:
I wonder if we could "fix" the `<svg>` tag in this situation, e.g. by replacing `<svg` with `<svg 
  xmlns="http://www.w3.org/2000/svg"`?

Reply:
Maybe, as far as my reading went, it seemed that this tag doesn't really do anything, but it's needed for browsers to accept the svg in the img src string. Normally embedded svgs are fine. I don't suspect that you could make an svg invalid by adding it if it doesn't have it already.